pom.xml

```diff
@@ -95,6 +95,11 @@
             <artifactId>a-vaadin-helper</artifactId>
             <version>1.6.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>collaboration-engine</artifactId>
+            <version>3.2.1</version>
```

Comment:
Could maybe have a comment in the code about why this is overridden?

Reply:
Done